SOURCES/0002-backport.patch

```diff
@@ -0,0 +1,360 @@
+--- a/igc.h	2022-09-23 12:17:01.000000000 +0000
++++ b/igc.h	2022-09-24 06:27:24.035810356 +0000
+@@ -217,6 +217,7 @@
+ 	struct timecounter tc;
+ 	struct timespec64 prev_ptp_time; /* Pre-reset PTP clock */
+ 	ktime_t ptp_reset_start; /* Reset time in clock mono */
++	char fw_version[32];
+ };
+ 
+ void igc_up(struct igc_adapter *adapter);
+--- a/igc_base.c	2022-09-23 12:17:01.000000000 +0000
++++ b/igc_base.c	2022-09-24 07:07:24.414598890 +0000
+@@ -205,6 +205,7 @@
+ 	case IGC_DEV_ID_I220_V:
+ 	case IGC_DEV_ID_I225_K:
+ 	case IGC_DEV_ID_I225_K2:
++	case IGC_DEV_ID_I226_K:
+ 	case IGC_DEV_ID_I225_LMVP:
+ 	case IGC_DEV_ID_I225_IT:
+ 	case IGC_DEV_ID_I226_LM:
+--- a/igc_defines.h	2022-09-23 12:17:01.000000000 +0000
++++ b/igc-module/igc_defines.h	2022-09-24 06:29:34.767001177 +0000
+@@ -160,6 +160,7 @@
+ #define IGC_NVM_RW_REG_START	1    /* Start operation */
+ #define IGC_NVM_RW_ADDR_SHIFT	2    /* Shift to the address bits */
+ #define IGC_NVM_POLL_READ	0    /* Flag for polling for read complete */
++#define IGC_NVM_DEV_STARTER	5    /* Dev_starter Version */
```

Comment:
What is the goal of this addition?

Reply:
To support showing the version from ethtool.